src/lender/coordinator.sol

```diff
@@ -120,6 +121,16 @@ contract EpochCoordinator is Ticker, Auth, FixedPoint  {
         (uint orderJuniorSupply, uint orderJuniorRedeem) = juniorTranche.closeEpoch();
         (uint orderSeniorSupply, uint orderSeniorRedeem) = seniorTranche.closeEpoch();
 
+        //  if no orders exist epoch can be executed without validation
+        if (orderSeniorRedeem == 0 && orderJuniorRedeem == 0 &&
+        orderSeniorSupply == 0 && orderJuniorSupply == 0) {
+
+            juniorTranche.epochUpdate(0, 0, 0, orderJuniorSupply, orderJuniorRedeem);
+            seniorTranche.epochUpdate(0, 0, 0, orderSeniorSupply, orderSeniorRedeem);
+            lastEpochExecuted = safeAdd(lastEpochExecuted, 1);
+            return;
```

Comment:
Even though there are no orders the capital might still need rebalancing (which only happens when execute epoch is called), due to the increase of the senior debt & the new NAV

Reply:
thats also true. We are trying to avoid to recalculating the NAV if its not necessary. 

We will refactor this part anyway with minimum epoch concept